lib/mongoid/association/relatable.rb

```diff
@@ -426,6 +463,45 @@ def create_extension!(&block)
       def default_inverse
         @default_inverse ||= klass.relations[inverse_klass.name.underscore]
       end
+
+      def namespace_hierarchy(mod)
```

Comment:
The semantics of this method seem non-trivial; I think it might be worth documenting what it does, maybe along with a simple example of what it would return.

Reply:
Thanks!